packages/techdocs-common/api-report.md

```diff
@@ -17,38 +17,42 @@ import { ScmIntegrationRegistry } from '@backstage/integration';
 import { UrlReader } from '@backstage/backend-common';
 import { Writable } from 'stream';
 
-// Warning: (ae-missing-release-tag) "DirectoryPreparer" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type DirectoryFactory = {
+  reader: UrlReader;
+};
+
+// @public
 export class DirectoryPreparer implements PreparerBase {
-  constructor(config: Config, _logger: Logger_2, reader: UrlReader);
-  // Warning: (ae-forgotten-export) The symbol "PreparerResponse" needs to be exported by the entry point index.d.ts
-  //
-  // (undocumented)
-  prepare(
-    entity: Entity,
-    options?: {
-      logger?: Logger_2;
-      etag?: string;
-    },
-  ): Promise<PreparerResponse>;
+  // @deprecated
+  constructor(config: Config, _logger: Logger_2 | null, reader: UrlReader);
+  static fromConfig(
+    config: Config,
+    options: DirectoryFactory,
+  ): DirectoryPreparer;
+  prepare(entity: Entity, options?: PreparerOptions): Promise<PreparerResponse>;
 }
 
-// Warning: (ae-missing-release-tag) "GeneratorBase" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type ETag = string;
+
+// @public
 export type GeneratorBase = {
   run(opts: GeneratorRunOptions): Promise<void>;
 };
 
-// Warning: (ae-missing-release-tag) "GeneratorBuilder" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
 // @public
 export type GeneratorBuilder = {
   register(protocol: SupportedGeneratorKey, generator: GeneratorBase): void;
   get(entity: Entity): GeneratorBase;
 };
 
+// @public
+export type GeneratorFactory = {
```

Comment:
Similar; maybe `GeneratorOptions`

Reply:
Renamed to `GeneratorOptions`, https://github.com/backstage/backstage/pull/9718/commits/e257a8f0f499672b3e696df53d0604aba9ddbf0b.